boards/px4/fmu-v6xrt/src/spi.cpp

```diff
@@ -41,47 +41,65 @@
 #include <drivers/drv_sensor.h>
 #include <nuttx/spi/spi.h>
 
-#include <px4_platform_common/px4_config.h>
-#include <px4_log.h>
+constexpr px4_spi_bus_all_hw_t px4_spi_buses_all_hw[BOARD_NUM_SPI_CFG_HW_VERSIONS] = {
+	initSPIHWVersion(V6XRT_00, {
+		initSPIBus(SPI::Bus::LPSPI1, {
+			initSPIDevice(DRV_IMU_DEVTYPE_ICM42688P, SPI::CS{GPIO::Port2, GPIO::Pin11}, SPI::DRDY{GPIO::Port3, GPIO::Pin19}), /* GPIO_EMC_B2_01 GPIO2_IO11, GPIO_AD_20, GPIO3_IO19 */
```

Comment:
SPI1 is a icm42686p in the production batch

Reply:
@PetervdPerk-NXP Good catch